### PR TITLE
Nyxborn Unicorn and Omo, Queen of Vesuva

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/nyxborn_unicorn.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Nyxborn Unicorn
+ManaCost:1 W
+Types:Enchantment Creature Unicorn
+PT:2/2
+K:Bestow:3 W
+K:Mentor
+S:Mode$ Continuous | Affected$ Card.EnchantedBy | AddPower$ 2 | AddToughness$ 2 | AddKeyword$ Mentor | Description$ Enchanted creature gets +2/+2 and has mentor.
+Oracle:Bestow {3}{W} (If you cast this card for its bestow cost, it's an Aura spell with enchant creature. It becomes a creature again if it's not attached to a creature.)\nMentor (Whenever this creature attacks, put a +1/+1 counter on target attacking creature with lesser power.)\nEnchanted creature gets +2/+2 and has mentor.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/nyxborn_unicorn.txt:8]: DeckHas:Ability%Counters

File: forge-game/src/main/java/forge/game/card/CounterEnumType.java [hunk 2]
```diff
@@ -163,7 +165,7 @@ public enum CounterEnumType {
 
     FLOOD("FLOOD", 0, 203, 255),
 
-    FORESHADOW("FRSHD",144,99, 207),
+    FORESHADOW("FRSHD",144, 99, 207),
```
Comment (Northmoc) [inline forge-game/src/main/java/forge/game/card/CounterEnumType.java:168]: might as well put the ` ,144` too while we're here 🤷

File: forge-gui/res/cardsfolder/upcoming/omo_queen_of_vesuva.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Name:Omo, Queen of Vesuva
+ManaCost:2 GU
+Types:Legendary Creature Shapeshifter Noble
+PT:1/5
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigPutCounter | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, put an everything counter on each of up to one target land and up to one target creature.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPutCounter | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, put an everything counter on each of up to one target land and up to one target creature.
+SVar:TrigPutCounter:DB$ PutCounter | TargetMin$ 0 | TargetMax$ 1 | ValidTgts$ Land | TgtPrompt$ Select target land | CounterType$ EVERYTHING | CounterNum$ 1 | SubAbility$ DBPutCounter
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/omo_queen_of_vesuva.txt:7]: Select up to one target land

File: forge-gui/res/cardsfolder/upcoming/omo_queen_of_vesuva.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Name:Omo, Queen of Vesuva
+ManaCost:2 GU
+Types:Legendary Creature Shapeshifter Noble
+PT:1/5
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigPutCounter | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, put an everything counter on each of up to one target land and up to one target creature.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPutCounter | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, put an everything counter on each of up to one target land and up to one target creature.
+SVar:TrigPutCounter:DB$ PutCounter | TargetMin$ 0 | TargetMax$ 1 | ValidTgts$ Land | TgtPrompt$ Select target land | CounterType$ EVERYTHING | CounterNum$ 1 | SubAbility$ DBPutCounter
+SVar:DBPutCounter:DB$ PutCounter | TargetMin$ 0 | TargetMax$ 1 | ValidTgts$ Creature | TgtPrompt$ Select target creature | CounterType$ EVERYTHING | CounterNum$ 1 | SubAbility$ DBPutCounter
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/omo_queen_of_vesuva.txt:8]: similar prompt
I think this SubAbility is bad, too

**Review by Hanmac — APPROVED** [review]

**Pull request merged**